src/main/java/redis/clients/jedis/CommandObjects.java

```diff
@@ -2195,6 +2195,14 @@ protected CommandArguments commandArguments(ProtocolCommand command) {
     return new CommandObject<>(args, BuilderFactory.STREAM_PENDING_ENTRY_LIST);
   }
 
+  public final CommandObject<List<StreamPendingEntry>> xpending(String key, String groupname,
+      String start, String end, int count, String consumername) {
```

Comment:
add `XpendingOption` in args

Reply:
It's problematic - because if the user not supplying `start` and `end` params, the `addParams` function adds the defaults '-' and '+'. But if the user does supply `start` and `end` as Strings, he can't use exclusive range.

Any ideas what to do?

Reply:
Is this OK？
```java
 /**
  * Should be used only with XGROUP CREATE  
  * 
  * <code>
   * XGROUP CREATE mystream consumer-group-name $
   * </code>
   */
  public static final StreamEntryID LAST_ENTRY = new StreamEntryID() {

    private static final long serialVersionUID = 1L;

    @Override
    public String toString() {
      return "$";
    }
  };
```